nifi-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -371,8 +371,7 @@ categorizing them by their functions.
 	the content fetched from HDFS.
 - *FetchS3Object*: Fetches the contents of an object from the Amazon Web Services (AWS) Simple Storage Service (S3). The outbound FlowFile contains the contents
   received from S3.
-- *GetKafka*: Consumes messages from Apache Kafka. The messages can be emitted as a FlowFile per message or can be batched together using a user-specified
-  delimiter.
+- *GetKafka*: Fetches messages from Apache Kafka, specifically for 0.8.x versions. The messages can be emitted as a FlowFile per message or can be batched together using a user-specified delimiter.
```

Comment:
It's my understanding that this is the best solution for all versions prior to and including `0.8.x`. I am not sure of compatibility with individual versions, as Kafka does not maintain backward compatibility on minor releases, but we don't have any other processors for `0.7.x`, etc.